CHANGELOG.asciidoc

```diff
@@ -35,6 +35,7 @@ https://github.com/elastic/apm-server/compare/71df0d96445df35afe27f38bcf734a0828
 - Support for `transaction.span_count.dropped.total` {pull}448[448].
 - Optional field `transaction.sampled` {pull}441[441]
 - Add Kibana sourcefilter for `sourcemap.sourcemap` {pull}454[454]
+- Document experimental sourcemap feature {pull}462[462]
```

Comment:
Not sure we should include documentation updates  as a separate item here. We already have:

> Support for sourcemap mapping on incoming frontend requests

generally, people will expect us to have documentation as part of a new feature.
We could add this PR to the item above:

 - Support for sourcemap mapping on incoming frontend requests {pull}381[381] {pull}462[462]